Formula/openjdk.rb

```diff
@@ -23,23 +23,30 @@ class Openjdk < Formula
   keg_only :shadowed_by_macos
 
   depends_on "autoconf" => :build
+  depends_on "pkg-config" => :build
   depends_on xcode: :build
+  depends_on "giflib"
+  depends_on "harfbuzz"
+  depends_on "jpeg-turbo"
+  depends_on "libpng"
+  depends_on "little-cms2"
   depends_on macos: :catalina
 
+  uses_from_macos "cups"
+  uses_from_macos "unzip"
+  uses_from_macos "zip"
+  uses_from_macos "zlib"
+
   on_linux do
-    depends_on "pkg-config" => :build
     depends_on "alsa-lib"
-    depends_on "cups"
     depends_on "fontconfig"
-    depends_on "gcc"
```

Comment:
Not getting picked up by the audit. Weird.

Reply:
On side note, this had indirectly linked `zstd` for `emscripten`. Should check on opportunistic linkage and whether it is worth adding as direct dependency:
```
Full linkage --cached --test --strict emscripten output
  Undeclared dependencies with linkage:
    zstd
```

Reply:
That's from the LLVM build, picked up via `gcc`. We should pass `-DLLVM_ENABLE_ZSTD=OFF` to the LLVM build. We also probably want to disable `libxml2`, and perhaps `ncurses`.

Reply:
Audit fixed in Homebrew/brew#13918.